presto-parser/src/main/java/com/facebook/presto/sql/tree/CreateTable.java

```diff
@@ -31,17 +31,17 @@
     private final List<Property> properties;
     private final Optional<String> comment;
 
-    public CreateTable(QualifiedName name, List<TableElement> elements, boolean notExists, List<Property> properties, Optional<String> comment)
+    public CreateTable(QualifiedName name, List<? extends TableElement> elements, boolean notExists, List<Property> properties, Optional<String> comment)
```

Comment:
why this change?

Reply:
By making these covariant, it simplified a test case (I can now define a test case where elements are ColumnDefinition, use them in the test case, and also pass them into this constructor which can now take in TableElement or any of TableElement's subtypes).